### ISSUE TITLE
Strange rendering/scrollable IDL boxes

### ISSUE THREAD

**Issue by marcoscaceres** [speced/bikeshed#2861]
When clicking on the definition for a IDL construct (e.g., interface) the container suddenly becomes scrollable and the references box goes gets culled:

![Screenshot 2024-05-23 at 10 16 30 AM](https://github.com/speced/bikeshed/assets/870154/5b3e9a87-7e88-4f65-933e-6000dcd6464f)

Try it here by clicking on  `CredentialRequestOptions`:
https://w3c.github.io/webappsec-credential-management/#credentialrequestoptions-dictionary

Worst case scenario, it end up like this: 
![Screenshot 2024-05-23 at 10 16 58 AM](https://github.com/speced/bikeshed/assets/870154/37290dc0-c46d-4f7b-a174-5a146d73f02c)

Expected the DFN box to be on top and not inside the IDL box.

**Comment by tabatkins** [comment]
You're looking at a spec that hasn't been regenerated in 3 years. The popup code has been significantly improved since then. ^_^

**Comment by tabatkins** [comment]
In particular, that source looks like this now:

![Untitled](https://github.com/speced/bikeshed/assets/682840/403af006-8605-4af4-bd62-3bc5ad75ff87)

**Comment by marcoscaceres** [comment]
Thanks for fixing that 🥰

**Comment by marcoscaceres** [comment]
Wait, are you sure? the spec was generated this week: 
![IMG_2295](https://github.com/speced/bikeshed/assets/870154/df1d2690-8e09-4b8a-9511-bec3ca06f5e3)

I’m still seeing it in the link above? And on TR, which was generated 23 May 2024. 

![IMG_2296](https://github.com/speced/bikeshed/assets/870154/72f40628-8e06-43d3-ae5b-bfa06da2a83a)

**Comment by tabatkins** [comment]
No, I generated it locally, and the dom structure isn't even compatible with that rendering - they're generated on the fly and as children of body, now. In the version you linked to they're all pregenerated, as children of the dfns, which is how I used to render them.

I guess they're using an old Bikeshed to do the generation.

**Comment by tabatkins** [comment]
Yeah, that structure was entirely changed as part of the 4.0 update https://github.com/speced/bikeshed/issues/1773#issuecomment-1847977011

But it looks like they're using specprod https://github.com/w3c/webappsec-credential-management/blob/main/.github/workflows/auto-publish.yml so I'm just *really* confused about this.

**Comment by marcoscaceres** [comment]
@sidvishnoi, any ideas on the Specprod side?

**Comment by sidvishnoi** [comment]
From latest action run there:

```
  $ pip3 --version
    pip 20.0.2 from /usr/lib/python3/dist-packages/pip (python 3.8)
                                                        ^^^^ whoaa
  $ pip3 install bikeshed --quiet
    ERROR: launchpadlib 1.10.13 requires testresources, which is not installed.
  $ bikeshed update
    Bringing data files up-to-date...
    ...
    Done!
  $ pip3 show bikeshed | grep -i version
    Version: 3.14.5
             ^^^^^ whoaa
```

That workflow is using ubuntu-20.04, instead of ubuntu-latest, so they end up using old python, which I think, in turn leads to old bikeshed being used.

**Comment by marcoscaceres** [comment]
Ok cool. I’ll get that fixed over in the other repo.

**Comment by tabatkins** [comment]
Phew, thanks for debugging this, it'll help a lot of people.

**Comment by marcoscaceres** [comment]
We should maybe do a search to see if other repos are affected. 

Should we keep this open as a reminder?

**Comment by inexorabletash** [comment]
FYI we just ran into this in [WICG/cookie-store](https://github.com/WICG/cookie-store/commit/663ddf47a189c65b36277020ab237a8513b6162e) and fortunately I remembered seeing this issue. I just  fixed it there and checked repos I've touched recently. I directly fixed, [WICG/entries-api](https://github.com/WICG/entries-api/commit/85956a1525b758433fca595702232aeaa42afaec), [WICG/local-font-access](https://github.com/WICG/local-font-access/commit/f54b2b7b7e560d1f12220495b115abec39ccba53), and [w3c/web-locks](https://github.com/w3c/web-locks/commit/c1afe5d1bebbea24634d3357ca60428102c7c84a) and filed PRs for [webmachinelearning/webnn](https://github.com/webmachinelearning/webnn/pull/704) and [w3c/window-management](https://github.com/w3c/window-management/pull/146) so... it's likely very widespread.

**Comment by marcoscaceres** [comment]
Reopening so we remember to do a search.

**Comment by sidvishnoi** [comment]
I can probably add a warning to spec-prod when using older Ubuntu.
